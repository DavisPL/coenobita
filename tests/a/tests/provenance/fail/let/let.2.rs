#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::provenance((root,root))]
    let w = 5;

    #[cnbt::provenance((*,root))]
    let x = w;

    #[cnbt::provenance((root,*))]
    let y = x;

    #[cnbt::provenance((*,*))]
    let z = w;
}