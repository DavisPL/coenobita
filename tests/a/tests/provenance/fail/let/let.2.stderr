error: expected (root,*), found (*,root)
  --> tests/provenance/fail/let/let.2.rs:12:13
   |
12 |     let y = x;
   |             ^
